README.md

```diff
@@ -1,10 +1,45 @@
 # Theia
 
-Theia contains Network Flow Visibility functions which are exacted from the
-[Antrea main repo](https://github.com/antrea-io/antrea). While flow exporter and
-flow aggregator are kept in the Antrea repo, other flow visibility modules, such
-as ClickHouse and Grafana related ones, will be moved to this new repo.
-
-During the code migration period, we will still keep related functions available
-and stable in the Antrea repo, and the functions in this new repo won't be ready
-until we announce the migration is completed.
+Theia is a network observability and analytics platform for Kubernetes. It is
+built on top of [Antrea](https://github.com/antrea-io/antrea), and consumes
+[network flows exported by Antrea](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md)
+to provide fine-grained visibility into the network communications in a
+Kubernetes cluster.
+
+## Getting Started
+
+Getting started with Theia is simple. You can follow the [Getting Started](docs/getting-started.md)
+guide to install Theia and start rocking!
+
+Theia supports network flow visualization and monitoring with Grafana. Check the
+[Network Flow Visibility](docs/network-flow-visibility.md) document to learn
+more.
+
+Based on the collected network flow information, Theia can recommend appropriate
+NetworkPolicy configuration to secure Kubernetes network and applications.
+Please refer to the [NetworkPolicy Recommendation](docs/networkpolicy-recommendation.md)
+user guide to learn more.
+
+## Contributing
+
+The Antrea community welcomes new contributors. We are waiting for your PRs!
+
+* Before contributing, please get familiar with our
+[Code of Conduct](CODE_OF_CONDUCT.md).
+* Check out the Antrea [Contributor Guide](CONTRIBUTING.md) for information
+about setting up your development environment and our contribution workflow.
+* Learn about Antrea's [Architecture and Design](https://github.com/antrea-io/antrea/blob/main/docs/design/architecture.md).
+Your feedback is more than welcome!
```

Comment:
This is not about this PR, but we also need an arch doc for Theia to integrate Antrea's one.